2021/01/12 19:53:55 [error] 29#29: *14 connect() failed (113: Host is unreachable) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "localhost:8088"
2021/01/12 19:54:03 [error] 29#29: *14 connect() failed (113: Host is unreachable) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "localhost:8088"
2021/01/12 19:54:28 [error] 29#29: *14 connect() failed (113: Host is unreachable) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "localhost:8088"
2021/01/12 19:55:24 [error] 29#29: *14 connect() failed (113: Host is unreachable) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "localhost:8088"
2021/01/12 19:56:13 [error] 29#29: *14 connect() failed (113: Host is unreachable) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "localhost:8088"
2021/01/12 19:58:11 [error] 29#29: *22 connect() failed (113: Host is unreachable) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "localhost:8088"
